pkgs/applications/networking/p2p/soulseekqt/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, fetchurl, qmake
+, makeWrapper
+, qtbase, dbus
+, expat, zlib, fontconfig
+, libpng, libX11, libxcb, libXau, libXdmcp, freetype, libbsd
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "soulseekqt-${version}";
+  version = "2016-1-17";
+
+  mainbin = "SoulseekQt-" + (version) +"-"+ (if stdenv.is64bit then "64bit" else "32bit");
+  src = fetchurl ({
```

Comment:
I think readability could be improved by moving some parts of this in the let block.